bot/cogs/admin.py

```diff
@@ -257,6 +257,38 @@ async def pog(self, ctx, *args):
             return
         await disp.WRONG_USAGE.send(ctx, ctx.command.name)
 
+    @commands.command()
+    @commands.guild_only()
+    async def accounts(self, ctx, *args):
+        if len(args) == 0:
+            await disp.ACC_ALL_HANDOUT.send(ctx, "enabled" if lobby.accounts_enabled() else "disabled")
+            return
+        arg = args[0]
+        if arg == "unlock":
+            if lobby.accounts_enabled():
+                await disp.ACC_ALL_HANDOUT.send(ctx, "already enabled")
+                return
+            lobby.set_lobby_accounts_enabled(True)
+            await disp.ACC_ALL_HANDOUT.send(ctx, "enabled")
+            return
+        if arg == "lock":
+            if not lobby.accounts_enabled():
+                await disp.ACC_ALL_HANDOUT.send(ctx, "already disabled")
+                return
+            lobby.set_lobby_accounts_enabled(False)
+            removed = []
+            for p_id in lobby.get_all_ids_in_lobby():
+                player = Player.get(p_id)
```

Comment:
All the player objects are already referenced in the lobby, I feel it would be cleaner to have a new function in the lobby module that would return a copy of the lobby list:
```py
def get_all_in_lobby():
    return _lobby_list.copy()
```
Then you could call this new function right away without having to get the objects from the cache and checking them

Reply:
Oops, not resolved.  Still new to Github 🙃.  I agree, I'll put that in!